public/ACMW/css/events.css

```diff
@@ -62,61 +53,33 @@ body {
 }
 .open .dropdown-toggle {
 	color: #fff;
-	background-color: #555;
+	background-color: #555 !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)